dev/make-distribution.sh

```diff
@@ -29,7 +29,7 @@ set -e
 set -x
 
 # Figure out where the Spark framework is installed
-SPARK_HOME="$(cd "`dirname "$0"`"; pwd)"
+SPARK_HOME="$(cd "`dirname "$0"`/.."; pwd)"
```

Comment:
Yeah, I'm even happy to not do this one. It just bugged me that this file was in the top level. It could be symlinked but that doesn't save much. Yeah I suspect some tools would need to look for it in both locations.